tests/jvm/src/test/resources/metacp.expect

```diff
@@ -36,73 +36,6 @@ _empty_.B#`<init>`(). => primaryctor <init>: (): B
 _empty_.B#a(). => def a: : A
   A => _empty_.A#
 
-com/javacp/ClassSuffix$Inner$Bar$Fuz.class
-------------------------------------------
```

Comment:
Nested classes are now stored in their owner's semanticdb files, right?

Reply:
Correct, these entries are included below see https://github.com/scalameta/scalameta/pull/1367/files#diff-026fd03a8437acdb4213e4e80bf077d1R71